convert/convert.php

```diff
@@ -188,24 +183,21 @@ function getTable($value, $from_unit, $to_unit, $precision) {
 
 
 	if (isset($_POST['from_unit']) && isset($_POST['value'])) {
-    	$_POST['value'] = $_POST['value'] + 0;
-
-
-		$o .= ($conv->getTable($_POST['value'], $_POST['from_unit'], $_POST['to_unit'], 5))."</p>";
+		$_POST['value'] = $_POST['value'] + 0;
+		$o .= ($conv->getTable($_POST['value'], $_POST['from_unit'], $_POST['to_unit'], 5)) . '</p>';
```

Comment:
```suggestion
		$o .= ($conv->getTable(intval($_POST['value']), $_POST['from_unit'], $_POST['to_unit'], 5)) . '</p>';
```

Reply:
The used class `UnitConvertor` is last edited from 2002 and therefore contains a deprecated constructor + CRLF. I added some type-hints where I was sure about it and added `@TODO` to it to have the need to rewrite it to a composer-based package.